zrml/prediction-markets/src/lib.rs

```diff
@@ -470,7 +470,7 @@ mod pallet {
             period: MarketPeriod<T::BlockNumber, MomentOf<T>>,
             metadata: MultiHash,
             market_type: MarketType,
-            dispute_mechanism: MarketDisputeMechanism<T::AccountId>,
```

Comment:
The front-end should be informed in the changelogs for devs file, because multiple extrinsic headers are changed.